client/components/Scenario/ContentSlide.jsx

```diff
@@ -60,25 +136,34 @@ class ContentSlide extends React.Component {
                         components={slide.components}
                         onResponseChange={onResponseChange}
                     />
-
-                    <Button.Group>
-                        <Button
-                            color="grey"
-                            onClick={this.props.onClickBack}
-                            content={'Back'}
-                        />
-                        <Button
-                            color="green"
-                            onClick={this.props.onClickNext}
-                            content={finishOrProceedLabel}
+                    <Popup
+                        content="Go back to the previous slide"
+                        trigger={
+                            <Button
+                                floated="left"
+                                color="grey"
+                                onClick={onClickBack}
+                                content={'Back'}
+                            />
+                        }
+                    />
+                    <Button.Group floated="right">
+                        <Popup
+                            content={fwdButtonTip}
+                            trigger={<Button {...fwdButtonProps} />}
                         />
                         {hasPrompt && (
                             <React.Fragment>
                                 <Button.Or />
-                                <Button
-                                    color="yellow"
-                                    onClick={this.onSkip}
-                                    content="Skip"
+                                <Popup
+                                    content="Skip these prompts and go to next slide."
+                                    trigger={
+                                        <Button
+                                            color="yellow"
+                                            onClick={onSkip}
+                                            content="Skip"
```

Comment:
Nit: what do you think of this button saying "Choose to Skip"? It's a little wordier, but I think it would help make it clearer why a question could be both required and skippable. Namely that we're asking them to make a choice.

Reply:
I'll change it to "Choose to Skip", but we should ask Garron or Justin what they expect to display here.